frontend_tests/node_tests/integration_bot_widget.js

```diff
@@ -0,0 +1,50 @@
+set_global('page_params', {
+    is_admin: false,
+    realm_users: [],
+});
+
+set_global('$', function () {
+});
+
+add_dependencies({
+    people: 'js/people.js',
+    stream_data: 'js/stream_data.js',
+});
+
+set_global('blueslip', {});
+
+var integration_bot_widget = require('js/integration_bot_widget.js');
+
+(function test_set_integration_bot_url() {
+    var external_api_uri_subdomain = "https://chat.zulip.org/api/v1/external/";
+    var integration_url = "airbrake";
+    var bot_full_name = "airbrake";
+    var bot_short_name = "airbrake";
+    var bot_avatar_file; // I am not sure how to point this variable to a file.
```

Comment:
Ah, I see. You can just mock the file, like make it a string or something. This is because you are going to be be mocking `channel.post`, so you can test exactly what is going on there.

You can look at `settings_org.js` tests for samples on `channel.post` I think.